src/core/unit_tests/rotation_test.cpp

```diff
@@ -189,6 +189,38 @@ BOOST_AUTO_TEST_CASE(propagate_omega_quat_particle_test) {
   }
 }
 
+BOOST_AUTO_TEST_CASE(convert_operator_body_to_space_test) {
+  auto constexpr sqrt_2_half = Utils::sqrt_2() / 2.0;
+  // rotation around z-axis by pi/2
```

Comment:
sorry that I am late but what does rotation around z-axis mean here?
I do not get the test

Reply:
It's a typo in the comment. The quaternion actually describes a rotation around the y-axis. This can be fixed as part of the 4.2.1 release spellcheck or as part of the fix for #4667. The idea behind this test is to make sure the old behavior is not re-introduced by mistake. As I understood it , the behavior in ticket #4667 is actually caused by two bugs: the rotation matrix was incorrectly applied (the particle quaternion describes the rotation from the z-axis to the director, so we need to revert the direction) and the prefactor was not transformed.